spec/requests/api/v1/transactions_request_spec.rb

```diff
@@ -0,0 +1,155 @@
+require "rails_helper"
+
+describe "transaction requests" do
+  it "returns data for all transactions" do 
+    create_list(:transaction, 3)
+
+    get "/api/v1/transactions"
+    transactions = JSON.parse(response.body)
+
+    expect(response).to be_successful
+    expect(transactions.count).to eq(3)
+    expect(transactions.first).to have_key("result")
+  end
+  it "returns a transaction given the id" do 
+    create(:transaction, id: 1)
+    create(:transaction)
+    create(:transaction)
+
+    get "/api/v1/transactions/1"
+
+    transaction = JSON.parse(response.body)
+    
+    expect(response).to be_successful
+    expect(transaction["id"]).to eq(1)
+  end
+  it "can find a transactions given the id" do 
+    create(:transaction, id: 1)
+    create(:transaction)
+    create(:transaction)
+
+     get "/api/v1/transactions/find?id=1"
+
+    transaction = JSON.parse(response.body)
+    expect(transaction["id"]).to eq(1)
+  end
+   it "can find all transactions given the id" do 
+    create(:transaction, id: 1)
+    create(:transaction)
+    create(:transaction)
+
+     get "/api/v1/transactions/find_all?id=1"
+
+    transaction = JSON.parse(response.body)
```

Comment:
Make sure you are consistent in your whitespace between the get and transaction.